ui/desktop/app/components/reveal/index.js

```diff
@@ -0,0 +1,7 @@
+import RoseDropdownComponent from 'rose/components/rose/dropdown';
+
+export default class RevealComponent extends RoseDropdownComponent {
+  constructor() {
```

Comment:
Is this necessary?

Reply:
I don't think so. I forgot to add a comment to ask you exactly this. Now I know lol thanks!!